content/post/2018-03-30-sct-encoding.markdown

```diff
@@ -0,0 +1,413 @@
+---
+author: Jacob Hoffman-Andrews, Boulder developer
+date: 2017-07-06T00:00:00Z
+excerpt: Here's how Signed Certificate Timestamps get embedded in certificates
+title: Technical deep dive: Encoding of SCTs in certificates
+slug: sct-encoding
+---
+
+Let's Encrypt recently [launched SCT embedding in
+certificates](https://community.letsencrypt.org/t/signed-certificate-timestamps-embedded-in-certificates/57187).
+This feature allows browsers to check that a certificate was submitted to a
+Certificate Transparency log. As part of the launch, we did a thorough review
+that the encoding of Signed Certificate Timestamps (SCTs) in our certificates
+matches the relevant specifications. In this post, I'll dive into the details.
+
+Certificate Transparency offers three ways to deliver SCTs to a browser: In a
+TLS extension, in stapled OCSP, or embedded in a certificate. We chose to
+implement the embedding method because it would just work for Let's Encrypt
+subscribers without additional work. In the SCT embedding method, we submit
+a "precertificate" with a poison extension to a set of CT logs, and gets back
```

Comment:
I think it would be helpful to briefly define what the poison extension is. (If you want to go into more detail, it would be an opportunity to mention ASN.1 NULL values, which some CAs botched in their pre-certificates.)